lib/elixir/pages/Writing Documentation.md

```diff
@@ -17,6 +17,7 @@ Documentation in Elixir is usually attached to module attributes. Let's see an e
       @moduledoc """
       This is the Hello module.
       """
+      @moduledoc since: "1.0.0"
```

Comment:
@fertapric i moved it up for completeness. wdty?

Reply:
You beat me by seconds, I was about to suggest the same! 👍